ext_mongodb.php

```diff
@@ -761,22 +883,73 @@ public function __toString() : string
 
 final class MaxKey implements Type, \Serializable
 {
-	use DenySerialization;
+	public function serialize() : string
+	{
+		return serialize( [] );
```

Comment:
Why not return an empty string here? "a:0:{}" serves no purpose if you're not going to inspect the argument to `unserialize()` at all. I'm not suggesting you change `unserialize()`, either.